src/pykeen/datasets/base.py

```diff
@@ -568,9 +568,10 @@ def __init__(
         if not self.path.is_file() and not self.url:
             raise ValueError(f'must specify url to download from since path does not exist: {self.path}')
 
-        self.relative_training_path = pathlib.PurePath(relative_training_path)
-        self.relative_testing_path = pathlib.PurePath(relative_testing_path)
-        self.relative_validation_path = pathlib.PurePath(relative_validation_path)
+        # relative paths within zipfile's always follow Posix path
+        self.relative_training_path = pathlib.PurePosixPath(relative_training_path)
```

Comment:
if you put a normal pathlib.Path into a pathlib.PurePosixPath, does it get converted?

Reply:
Seems to work, also for pure paths from a different OS:
```python-console
>>> pathlib.PurePosixPath(pathlib.Path("data", "test"))
PurePosixPath('data/test')
>>> pathlib.PureWindowsPath(pathlib.Path("data", "test"))
PureWindowsPath('data/test')
```

Reply:
However, our changes happened concurrent, and I like your change better than this.